Neos.Flow/Classes/ResourceManagement/ResourceTypeConverter.php

```diff
@@ -237,7 +247,7 @@ protected function handleHashAndData(array $source, PropertyMappingConfiguration
                 return $resource;
             }
 
-            if ($configuration->getConfigurationValue(ResourceTypeConverter::class, self::CONFIGURATION_IDENTITY_CREATION_ALLOWED) !== true) {
+            if ($configuration === null || $configuration->getConfigurationValue(ResourceTypeConverter::class, self::CONFIGURATION_IDENTITY_CREATION_ALLOWED) !== true) {
```

Comment:
Note: Maybe a missing configuration should not allow identity creation for the resource?

Reply:
Right now that is the behaviour, right? So changing it would make this a breaking change?

Reply:
Well, right now if no configuration is given it just fails miserably. It's not a case that will happen through the property mapper (that one requires a configuration), only if typeconverters are invoked manually.

Reply:
Ok, in that case I am all for "deny unless explicitly configured".

Reply:
```suggestion
            if ($configuration !== null && $configuration->getConfigurationValue(ResourceTypeConverter::class, self::CONFIGURATION_IDENTITY_CREATION_ALLOWED) !== true) {
```

Reply:
Fixed that separately with #2349